Tasks/RunPowerShellOnTargetMachines/RunPowerShellOnTargetMachines.ps1

```diff
@@ -2,21 +2,22 @@ param (
     [string]$environmentName,
     [string]$machineNames, 
     [string]$sourcePackage,
-    [string]$applicationPath,
     [string]$scriptPath,
     [string]$scriptArguments,
-    [string]$initializationScriptPath
+    [string]$initializationScriptPath,
+    [string]$runPowershellInParallel
     )
 
 Write-Verbose "Entering script RemotePowerShellRunner.ps1" -Verbose
 Write-Verbose "environmentName = $environmentName" -Verbose
 Write-Verbose "machineNames = $machineNames" -Verbose
 Write-Verbose "sourcePackage = $sourcePackage" -Verbose
-Write-Verbose "applicationPath = $applicationPath" -Verbose
 Write-Verbose "scriptPath = $scriptPath" -Verbose
 Write-Verbose "scriptArguments = $scriptArguments" -Verbose
 Write-Verbose "initializationScriptPath = $initializationScriptPath" -Verbose
 
+. ./RunPowerShellJob.ps1
+
 function Output-ResponseLogs
```

Comment:
We should have this function in a common module, as we are duplicating this now in many tasks. Just doesn't seem right to me.